### ISSUE TITLE
command announcement for all devices

### ISSUE THREAD

**Issue by amiga4everyone** [Apollon77/ioBroker.alexa2#788]
**Describe the bug**  
when i type an announcement des announcement coms from all echo devices but one. When ich type the announcement direkt in the commands.announcemnet field for this device i can hear it.

**To Reproduce**  
1. Type "Test" in Commands.announcement for all devices
2. hear "Test" from all echo devices but one (I have 4 devices)

**Expected behavior**  
I expect to hear it from all 4 devices

**Versions:**  
 - Adapter version:  3.11.2
 - JS-Controller version: 3.3.20
 - Node version: 12.22.7
 - Operating system: Linux (Container from buanet)

**Comment by Apollon77** [comment]
Please enable Debug log. Then check log for "Rate limit exceeded" info ... if it is in then it is a duplicate of #510 ..

**Comment by amiga4everyone** [comment]
I enabled the debug log and restarted the adapter. The log can be found in the attachment. I have sended again an output (seperately) on the Devices alexa2.0.Echo-Devices.G090XG12137409LQ (Büro) and alexa2.0.Echo-Devices.G000MW0474750BVW (WZ). These annoucments reached the devices. An output to alexa2.0.Echo-Devices.ddafe88c61db4369a82eb7114911b943 (überall) arrived only on the second device alexa2.0.Echo-Devices.G000MW0474750BVW. The announcment to all devices reached also all other devices but alexa2.0.Echo-Devices.G090XG12137409LQ (Büro). A rate limit exceeded does not appear in the log.
[Microsoft Excel-Arbeitsblatt (neu).csv](https://github.com/Apollon77/ioBroker.alexa2/files/7619572/Microsoft.Excel-Arbeitsblatt.neu.csv)
